src/test/java/org/jitsi/meet/test/MalleusJitsificus.java

```diff
@@ -278,8 +318,38 @@ public void run()
                 _url.appendConfig("config.startWithAudioMuted=true");
             }
 
+            if (region != null)
+            {
+                _url.appendConfig("config.deploymentInfo.userRegion=\"" + region + "\"");
+            }
+        }
+
+        public void start(ScheduledExecutorService pool)
+        {
+            this.pool = pool;
+
+            started = pool.schedule(this::join, joinDelayMs, TimeUnit.MILLISECONDS);
+            complete = pool.schedule(this::finish, joinDelayMs + durationMs, TimeUnit.MILLISECONDS);
+
+            if (enableFailureDetection)
+            {
+                long healthCheckIntervalMs = 5000; // Configure this?
```

Comment:
The default value is good enough for our use-case and we wouldn't want to tweak it but I agree it would be nice to have this configurable just because arbitrary constants mixed in the code are rarely a good idea.